ajo/PersonalizationPayloads.md

```diff
@@ -0,0 +1,396 @@
+## Personalization Payloads
+
+### Proposition Fetch
```

Comment:
I'd label the sections using the same terminology used in the Flow Diagrams so users can see how the payloads correspond to each step in the Flow Diagram. So this could be labeled "Interact Request - Fetch Personalization Content".

Reply:
Modified the labels to,

### Interact Request - Fetch Personalization Content
### Return Propositions - Personalization Content
### Send display notification(s)
### Send interact notification(s)